archive/tar.go

```diff
@@ -497,18 +502,32 @@ type ChangeWriter struct {
 	addedDirs map[string]struct{}
 }
 
+// ChangeWriterOpt can be specified in NewChangeWriter.
+type ChangeWriterOpt func(cw *ChangeWriter)
+
+// WithWhiteoutT sets the whiteout timestamp.
+func WithWhiteoutT(tm time.Time) ChangeWriterOpt {
```

Comment:
Should this be `WithWhiteoutTime` instead of `T`?

Reply:
Fixed